schema/loader_test.go

```diff
@@ -0,0 +1,120 @@
+package schema
+
+import (
+	"testing"
+
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/jarcoal/httpmock"
+	"github.com/santhosh-tekuri/jsonschema/v5"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCachingLoader(t *testing.T) {
```

Comment:
Consider splitting this large test into multiple smaller unit tests that exercise a single behavior.

Reply:
split into 2